### PR TITLE
Frontend setup

### PR DESCRIPTION
**Pull request by colbycheeze**
addresses #144

### CODE REVIEW

**Comment by rvennam** [comment]
Excited about pulling this in and start playing with it!

**Comment by colbycheeze** [comment]
@rvennam Yea we just got a new FED on the team so I'll be doing my best to teach her things and I can help you with anything that you might be curious about since there is a good amount going on (if you wanted to understand EVERYTHING, although you should be able to pretty quick just build new components or functionality)

File: client-src/README.md [hunk 1]
```diff
@@ -0,0 +1,275 @@
+# Logistics Wizard Demo
+
+[![Build Status](https://travis-ci.org/davezuko/react-redux-starter-kit.svg?branch=master)](https://travis-ci.org/davezuko/react-redux-starter-kit?branch=master)
```
Comment (l2fprod) [inline client-src/README.md]: @colbycheeze 

these come from the starter kit, we should update them to our build or just remove them.

Comment (colbycheeze) [inline client-src/README.md]: Yea I have been updating the readme as I go, just forgot about that section. I have a lot more changes locally I'll push up on next merge. Thanks for catching this one.

File: client-src/README.md [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+
+## Deployment
+Out of the box, this starter kit is deployable by serving the `~/dist` folder generated by `npm run deploy` (make sure to specify your target `NODE_ENV` as well). This project does not concern itself with the details of server-side rendering or API structure, since that demands an opinionated structure that makes it difficult to extend the starter kit. However, if you do need help with more advanced deployment strategies, here are a few tips:
+
+### Static Deployments
+If you are serving the application via a web server such as nginx, make sure to direct incoming routes to the root `~/dist/index.html` file and let react-router take care of the rest. If you are unsure of how to do this, you might find [this documentation](https://github.com/reactjs/react-router/blob/master/docs/guides/Histories.md#configuring-your-server) helpful. The Koa server that comes with the starter kit is able to be extended to serve as an API or whatever else you need, but that's entirely up to you.
+
+### Heroku
```
Comment (l2fprod) [inline client-src/README.md]: @colbycheeze 

We should not need this. Or adapted to Bluemix.

File: client-src/package.json [hunk 3]
```diff
@@ -72,9 +72,9 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/davezuko/react-redux-starter-kit.git"
+    "url": "git+https://github.com/IBM-Bluemix/logistics-wizard.git"
   },
-  "author": "David Zukowski <david@zuko.me> (http://zuko.me)",
+  "author": "Colby Williams <colbycheeze@us.ibm.com> (http://colbycheeze.com)",
   "license": "MIT",
```
Comment (l2fprod) [inline client-src/package.json]: @colbycheeze 

Logistics Wizard is licensed under Apache 2.0

**Pull request merged**